### ISSUE TITLE
error on 'exec selected lines' w/o selection

### ISSUE THREAD

**Issue by Alexey-T** [halfbrained/cuda_terminal_plus#23]
```
Init: cudatext
Init: cuda_addonman
Traceback (most recent call last):
  File "/home/user/.config/cudatext/py/cuda_terminal_plus/__init__.py", line 1805, in cmd_exec_selected
    self.termbar.run_cmd(CMD_EXEC_SEL)
  File "/home/user/.config/cudatext/py/cuda_terminal_plus/__init__.py", line 1001, in run_cmd
    caret_x,caret_y = caret[0:2]
ValueError: not enough values to unpack (expected 2, got 1)
```

**Comment by Alexey-T** [comment]
repro:
open new file, text
```
echo aa
echo 2
```
add terminal.
w/o selection, call SUBJ.

**Comment by halfbrained** [comment]
fixed (de269976d6a25f80881fbee0adbf376c4d25e8d5)
